MaplyAdapterTests/OSTileSourceFactoryTests.m

```diff
@@ -0,0 +1,37 @@
+//
+//  OSTileSourceFactoryTests.m
+//  MaplyAdapter
+//
+//  Created by David Haynes on 19/03/2016.
+//  Copyright © 2016 Ordnance Survey. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+@import MaplyAdapter;
+@import Expecta;
+#import "MaplyComponent.h"
+
+@interface OSTileSourceFactoryTests : XCTestCase
+
+@end
+
+@implementation OSTileSourceFactoryTests
+
+- (void)test27700TilesourceCreation {
+    MaplyQuadImageTilesLayer *testLayer = [OSTileSourceFactory create27700TileLayer];
+    expect(testLayer).toNot.beNil();
+    expect(testLayer.coverPoles).to.beFalsy();
+    expect(testLayer.drawPriority).to.equal(kMaplyImageLayerDrawPriorityDefault);
+    expect(testLayer.tileSource).toNot.beNil();
```

Comment:
Future area of improvement would be to test this more thoroughly.

In particular, the responsibilities that should be tested are:
* Setting the correct layer name in the base URL.
* Setting the correct SRS in the base URL.
* Seeting the correct service in the base URL.
* Setting the correct min/max zoom levels.
* Setting the correct coordinate reference system in the tileSource.
* Creating a suitable cache directory location.